content/blog/2023/web-pwa-native-RWA1.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Part 1 - Responsive Web App vs PWA vs Native Mobile App
+sub: When to choose a responsive web app (RWA) for your digital product
```

Comment:
@SondraE  I don't think Ive ever seen it written as a RWA. I don't think I would've known what that was if I just saw it in a list, lol. TIL...